### PR TITLE
Document Latest Features from Q4

### PR DESCRIPTION
**Pull request by ndneighbor**
Went through and documented features in platform from past feature releases. Added the following ->

- Log Filtering
- Discord Integration
- Beta Program
- Webhook Crash Notifcations
- Custom Start Commands
- Updatable Starters

Metro will be documented in a separate PR and transition doc.

### CODE REVIEW

File: src/pages/deploy/builds.md [hunk 1]
```diff
@@ -85,3 +85,7 @@ You can opt-in to using this new builder by enabling it in your project's deploy
             quality={80} />
 
 Please note that this is still in development and may not be 100% backwards compatible with the previous builder. If you have any thoughts/concerns/questions/feedback, [we would love to hear from you](https://discord.gg/xAm2w6g)!
+
+## Custom Start Commands
+
+You may define a custom start command in the Deployment Settings. This means you can deploy the same repository within Railway projects and start a different monorepo component for each one. [You can also see our documentation about Monorepos for more information.](/deploy/monorepo)
```
Comment (soggycactus) [inline src/pages/deploy/builds.md:91]: I'm going to update this once I merge my changes to give a little more detail on how start commands work

File: src/pages/diagnose/webhooks.md [hunk 2]
```diff
@@ -56,4 +56,8 @@ Slack supports integrating directly with webhooks.
 1. Enable incoming webhooks for your Slack instance (Tutorial [here](https://api.slack.com/messaging/webhooks#enable_webhooks))
 2. Get a hooks.slack.com address for your channel (Tutorial [here](https://api.slack.com/messaging/webhooks#create_a_webhook))
 3. Open up Railway, navigate to your project. Under Deployments -> Settings -> Webhooks, paste your URL
-4. Click the checkmark to save
\ No newline at end of file
+4. Click the checkmark to save
+
+## Crash Notifications
+
+Railway projects now allow you to pipe out project crashes via the configured webhook.
```
Comment (JakeCooper) [inline src/pages/diagnose/webhooks.md:63]: I'm unsure if we necessarily need this section, especially cause the events we emit might change later

```suggestion
Railway projects now allows you to pipe out project crashes via the configured webhook.
```

File: src/pages/deploy/deployments.md [hunk 3]
```diff
@@ -102,4 +110,10 @@ deployment in progress.
 
 If a deployment is completed, you can delete a live deploy by clicking the the
 three dots at the end of the deployment tab and select Remove. This will remove
-the deployment and stop any further project usage. 
+the deployment and stop any further project usage.
+
+### Deployment Restarts
```
Comment (soggycactus) [inline src/pages/deploy/deployments.md:115]: see my comment below - I think this section should probably just be "What happens if my deployment crashes?" or something like that

**Review by soggycactus — APPROVED** [review]

**Review by JakeCooper — APPROVED** [review]

File: src/pages/deploy/builds.md (hunk 1 above)
Comment (ndneighbor) [reply src/pages/deploy/builds.md:91]: Cool: should I wait to merge for your changes?

File: src/pages/deploy/deployments.md (hunk 3 above)
Comment (ndneighbor) [reply src/pages/deploy/deployments.md]: Agree- made the requisite changes and added a photo.

File: src/pages/diagnose/webhooks.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (ndneighbor) [reply src/pages/diagnose/webhooks.md]: Removed and fleshed out the deployments section instead.

**Pull request merged**